x-pack/plugin/ml/qa/single-node-tests/src/javaRestTest/java/org/elasticsearch/xpack/ml/integration/LearningToRankRescorerIT.java

```diff
@@ -83,7 +83,7 @@ public void testLearningToRankRescoreWithExplain() throws Exception {
               }
             }""");
         var response = client().performRequest(request);
-        assertExplainExtractedFeatures(response, List.of("type_tv", "cost", "two"));
+        assertExplainExtractedFeatures(response, List.of("cost", "type_tv", "two"));
```

Comment:
ℹ️ The bug is here: we should check the order of feature in explain is the same as the one in the model definition.